conf/application.conf

```diff
@@ -19,26 +33,42 @@ play.modules.enabled += "uk.gov.hmrc.play.bootstrap.graphite.GraphiteMetricsModu
 
 # Provides an implementation and configures all filters required by a Platform frontend microservice.
 play.modules.enabled += "uk.gov.hmrc.play.bootstrap.frontend.FrontendModule"
+play.http.filters = "uk.gov.hmrc.play.bootstrap.frontend.filters.FrontendFilters"
 
 # Default http client
 play.modules.enabled += "uk.gov.hmrc.play.bootstrap.HttpClientModule"
+play.i18n.langs = ["en", "cy"]
 
 # Custom error handler
-play.http.errorHandler = "uk.gov.hmrc.incometaxemploymentfrontend.config.ErrorHandler"
+play.http.errorHandler = "config.ErrorHandler"
 
-play.filters.headers.contentSecurityPolicy = "default-src 'self' 'unsafe-inline' localhost:9000 localhost:9032 www.google-analytics.com data:"
+play.filters.headers.contentSecurityPolicy = "default-src 'self' 'unsafe-inline' localhost:9000 localhost:9032 localhost:12345 www.google-analytics.com www.googletagmanager.com data:"
 
 # Play Modules
 # ~~~~
 # Additional play modules can be added here
+play.modules.enabled += "config.Modules"
+play.modules.enabled += "uk.gov.hmrc.play.bootstrap.AuthModule"
+
+# Secret key
+# ~~~~~
+# The secret key is used to secure cryptographics functions.
+# If you deploy your application to several instances be sure to use the same key!
+# Not set here so that MDTP frontends share the same secret key in the local environment
+# (see common.conf in frontend-bootstrap).
+# In server environments the secret comes from app-config-common
+# play.http.secret.key="NWCsJFOz4nRxQbYsUjUgzCzsiT5s9KNfiCfUDjWMDvNqCgHBSzNYwI8P7huPYGCa"
 
 microservice {
+
+  url = "http://localhost:9317"
+
   metrics {
     graphite {
       host = localhost
       port = 2003
       prefix = play.${appName}.
-      enabled = true
+      enabled = false
```

Comment:
should be true by default

Reply:
Changed, must of forgot to revert it after running tests